### ISSUE TITLE
Can it show comment information when typing props?

### ISSUE THREAD

**Issue by joy-yu** [vuejs/language-tools#4663]
### What problem does this feature solve?

There are no additional information to display when typing props.

![image](https://github.com/user-attachments/assets/1bff9b78-2b84-4191-9b85-2c7d2adb5530)

Is that possible to show extra information just like data in current file.
![image](https://github.com/user-attachments/assets/518761ef-1099-4ba9-8871-e5808ecc7919)

### What does the proposed solution look like?

No solution.

**Comment by joy-yu** [comment]
I studied the source code and implemented this function simply.

I found that the `CompletionItem` documentation only support markdown style, so we must transfer comment and jsdoc info to markdown manually.

Perhaps there are still many undiscovered edge cases. Is the official interested in fully implementing and test it?

![image](https://github.com/user-attachments/assets/0832f7b2-1fa5-4a76-b44d-eda6a349d313)
